src/nanotron/trainer.py

```diff
@@ -855,22 +888,42 @@ def setup_log_writers(
         return loggerwriter
 
     def pre_save_checkpoint(self) -> Path:
+
+        if wandb is not None and dist.get_rank(self.parallel_context.dp_pg) == 0:
+            if self.config.general.wandb_id is None:
+                self.config.general.wandb_id = wandb.run.id
+                self.config.general.wandb_project = wandb.run.project
```

Comment:
why don't we keep it as the default in the initial config?

Reply:
was because we previously need to store the wandb.run.id (and project) to pass it to lighteval to save the eval on the same wandb run. But we don't save to wandb anymore so no need for this will remove it thanks for spotting it!